auto_dev/protocols/protodantic.py

```diff
@@ -97,11 +98,20 @@ def create(
     message_import_path = ".".join(models_import_path.split(".")[:-1]) or "."
     messages_pb2 = pb2_path.with_suffix("").name
 
+    generated_strategies = strategies_template.render(
+        float_primitives=float_primitives,
+        integer_primitives=integer_primitives,
+        primitives_import_path=primitives_import_path,
+    )
+    strategies_outpath = test_outpath.parent / "primitive_strategies.py"
+    strategies_outpath.write_text(generated_strategies)
+
+    strategies_import_path = _compute_import_path(strategies_outpath, repo_root)
     tests = generated_tests = hypothesis_template.render(
         file=file,
         float_primitives=float_primitives,
         integer_primitives=integer_primitives,
-        primitives_import_path=primitives_import_path,
+        strategies_import_path=strategies_import_path,
```

Comment:
need to import from here such that the strategies that get registered in this module are available in the hypothesis caching mechanism; it won't know how to generate a strategy using `st.from_type` without.